content/docs/2.1/scalers/aws-cloudwatch.md

```diff
@@ -30,6 +31,12 @@ triggers:
     # Optional: AWS Secret Access Key, can use TriggerAuthentication as well
     awsSecretAccessKeyFromEnv: AWS_SECRET_ACCESS_KEY # default AWS_SECRET_ACCESS_KEY
     identityOwner: pod | operator # Optional. Default: pod
+    # Optional: defaultMetricCollectionTime
```

Comment:
Can you please provide some docs on these new parameters in our parameter list please?

For me it's not clear what they represent.

Reply:
Updated Parameter list in the doc. Can you please review, @tomkerkhove ?